rules-tests/CodingStyle/Rector/Closure/StaticClosureRector/FixtureAutoImport/anonymous_class_suppress_warning_doc.php.inc

```diff
@@ -1,39 +0,0 @@
-<?php
-
-namespace Rector\Tests\CodingStyle\Rector\Closure\StaticClosureRector\FixtureAutoImport;
-
-return new class {
-    /**
-     * Run the migrations.
-     *
-     * @SuppressWarnings(PHPMD.ElseExpression)
-     */
-    public function up(): void
```

Comment:
No, this test was used on crash on `@SuppressWarning` usage

Reply:
see:

- https://github.com/rectorphp/rector-src/pull/2808

Reply:
That's unrelated to StaticClosureRector. Should be in the place it crashes.

Reply:
I don't think this will regress as anonymous check is covered now, so we can skip that :)

Reply:
Thats only happen on auto import + StaticClosureRector + `@Suppresswarning` usage